### PR TITLE
nadsatt/movie-seat-booking

### PR DESCRIPTION
**Pull request by nadsatt**
I'm improving this app, please do not review it now!

[demo :(](https://nadsatt.github.io/kottans-homeworks/movie-seat-booking/)

### CODE REVIEW

File: submissions/nadsatt/index.html [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+  <main class="main">
+    <section class="movie-info">
+      <h3 class="movie-info__name">Baby driver</h3>
```
Comment (A-Ostrovnyy) [inline submissions/nadsatt/index.html]: Use `h1` for main page title.

File: submissions/nadsatt/index.html [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+            </label>
+          </li>
+        </ol>
+      </fieldset>
+      <fieldset class="control seat-control">
+        <legend class="control__name">Seeats</legend>
+        <ol class="control__check-list">
+          <li class="control__check-item ">
+            <label class="control__check">
+              <input class="control__check-input control__input" type="checkbox" name="1" disabled>
```
Comment (A-Ostrovnyy) [inline submissions/nadsatt/index.html]: Add and fill attribute `value` to the input.

File: submissions/nadsatt/index.html [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+          </li>
+          <li class="control__check-item">
+            <label class="control__check">
+              <input class="control__check-input control__input" type="checkbox" name="16" disabled>
+              <span class="control__check-box control__box"></span>
+            </label>
+          </li>
+          <li class="control__check-item">
+            <label class="control__check">
+              <input class="control__check-input control__input" type="checkbox" name="17">
```
Comment (A-Ostrovnyy) [inline submissions/nadsatt/index.html]: Add styles on `:hover`, and `:active` states.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Eye candy work!
See my comments below.

**Comment by nadsatt** [comment]
@A-Ostrovnyy thank you for review!

What was done:
- improved html & css code (core idea and html skeleton remained the same)
- added fixes according to your comments

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**